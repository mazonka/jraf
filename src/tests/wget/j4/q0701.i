command=jw login a@b.c http://server/