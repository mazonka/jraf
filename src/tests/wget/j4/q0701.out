JRAF_FAIL no users
==
